client/src/modules/user/UserProfile/EmailNotifications.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { PureComponent } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { saveEmailNotificationSettings } from '../model/actions';
+import DaySelector from './DaySelector';
+import './EmailNotification.scss';
+import { EmailNotificationsFrequency } from 'common/constants/enums';
+import { UserPropType } from 'common/constants/propTypes';
+
+const emailNotificationsOptions = [
+  { message: 'common.monday', value: EmailNotificationsFrequency.MONDAY },
+  { message: 'common.tuesday', value: EmailNotificationsFrequency.TUESDAY },
+  {
+    message: 'common.wednesday',
+    value: EmailNotificationsFrequency.WEDNESDAY
+  },
+  {
+    message: 'common.thursday',
+    value: EmailNotificationsFrequency.THURSDAY
+  },
+  { message: 'common.friday', value: EmailNotificationsFrequency.FRIDAY },
+  {
+    message: 'common.saturday',
+    value: EmailNotificationsFrequency.SATURDAY
+  },
+  { message: 'common.sunday', value: EmailNotificationsFrequency.SUNDAY }
+];
+
+class EmailNotifications extends PureComponent {
```

Comment:
`SummaryReportSettings`? This component is not an email notification it allows the user to set up sending reports frequency...

PS. Or if we stay with notification `NotificationsSettings`